data/binding/comparator_helper.go

```diff
@@ -11,3 +15,7 @@ func compareURI(v1, v2 fyne.URI) bool {
 	}
 	return v1.String() == v2.String()
 }
+
+func compareBytes(v1, v2 []byte) bool {
+	return bytes.Equal(v1, v2)
+}
```

Comment:
I saw why this was added and then fixed thanks to Staticcheck. However, do we really need this wrapper function after that?

Reply:
Good catch, thanks